status-go-version.json

```diff
@@ -3,7 +3,7 @@
     "_comment": "Instead use: scripts/update-status-go.sh <rev>",
     "owner": "status-im",
     "repo": "status-go",
-    "version": "v0.151.5",
-    "commit-sha1": "31d02f5197684317ad9f14874b0980d5ca716be6",
-    "src-sha256": "0ih77pq3lbk6qadv018pp7gzd6vhgzcl34vm1vg2mdmv2f15zvlw"
+    "version": "ios/push-notifications",
```

Comment:
Is this ok? It doesn't need to be v0.151.6?

Reply:
until it's tested and ready to merge, we set it to branch, we create a tag/version only after it's merged on status-go, then we update here before merging